### DISCUSSION TITLE
Remapping Snack toggle

### DISCUSSION THREAD

**Discussion by EricDriussi** [LazyVim/LazyVim#4969] [Q&A] [answered]
For some reason I can't seem to do something like this in `keymaps.lua`:

```lua
Snacks.toggle.option("relativenumber", { name = "Relative Number" }):map("<leader>n")
```

I can see the default `<leader>uL` mapping but not mine.

I use to have this before v13:

```lua
LazyVim.toggle.map("<leader>n", LazyVim.toggle("relativenumber", { name = "Relative Number" }))
```

What am I missing here?

**Comment by dpetka2001** [comment, marked as answer]
`<leader>n` is defined in `keys` of `snacks.nvim` for showing the notification history, so you would have to disable that mapping as well for your own to take effect.

**Reply by EricDriussi** [reply]
oh well that makes a lot of sense! Indeed something like this did the job, thanks!

```lua
{
	"folke/snacks.nvim",
		keys = {
		{ "<leader>n", false },
	},
},
```
